src/components/EventBanner/EventBanner.jsx

```diff
@@ -1,26 +1,25 @@
 import React from "react";
 import "./EventBanner.scss";
+import dayjs from 'dayjs';
 
-export default function EventBanner({flyerSource, altText, title, desc, endDate, meetingLink}) {
-    let flyer = require(`../../assets/flyers/${flyerSource}`)
-    let imageElement;
-
-    if (altText) {
-        imageElement = <img src={flyer.default} alt={`${altText}`}/>
-    } else {
-        imageElement = <img src={flyer.default} alt={`Flyer for ${title} event`}/>
-    }
+export default function EventBanner({flyerSource, altText, title, desc, startDate, endDate, meetingLink, isGbm, now=dayjs()}) {
+    let flyer = flyerSource ? require(`../../assets/flyers/${flyerSource}`) : null;
+    let imageElement = flyer ? <img loading="lazy" src={flyer.default} alt={altText ? `${altText}` : `Flyer for ${title ? `${title} ` : ""}event`}/> : null;
 
+    let inProgress = startDate && endDate && startDate.isBefore(now) && endDate.isAfter(now);
 
-    if (endDate === undefined || new Date().getTime() < endDate.getTime()) {
+    console.log("EVENT", title, isGbm, now.toString(), startDate ? startDate.subtract(7, 'day').toString() : startDate);
+    if ((!endDate || now.isBefore(endDate)) && (!startDate || !isGbm || startDate.subtract(7, 'day').isBefore(now))) {
         return(
             <div className="flyer">
+                {startDate ? startDate.format("MMM. D, h:mm a") : null}
```

Comment:
I think this looks a little odd outside of the actual borders of the banner. What if we put it in the top-right corner instead?

Reply:
I was kinda avoiding that because I didn't want to overhaul the event's appearance too much. I was thinking it would be easier to do in a separate PR

Reply:
I was able to move it without touching css